html/php-pre-challenge3/index.php

```diff
@@ -4,3 +4,40 @@
 $dsn = 'mysql:dbname=test;host=mysql';
 $dbuser = 'test';
 $dbpassword = 'test';
+
+try {
+  $db = new PDO ($dsn , $dbuser , $dbpassword);
+} catch (PDOException $e) {
+  http_response_code(500);
+  exit();
+}
+
+// データベースから数値を取り出す
+$numbers = $db -> query('SELECT * FROM prechallenge3');
+$i = 0;
+while($number = $numbers -> fetch()){
+  $num[$i] = $number['value'];
+  $i++;
+}
```

Comment:
select文でワイルドカード*を使うのはやめましょう。DBから取り出す時点で単独でlimitより大きい値はメモリの無駄なので取り出さないようにしましょう。見栄えを良くするために昇順ソートもかけましょう。

また、phpで単純に配列に詰め込む処理では添え字 `$i` の指定は不要です。

今回は修正不要ですが、配列変数は複数形の名前がよいですね(×num, 〇nums)

まとめると下記のコードのようにしましょう。コピペOKです。

```suggestion
$stmt = $db->prepare('SELECT value FROM prechallenge3 where (value <= :limit) order by value asc');
$stmt->bindValue(':limit', $limit, PDO::PARAM_INT);
$stmt->execute();
while ($number = $stmt->fetch()) {
  $num[] = $number['value'];
}
```